### DISCUSSION TITLE
GitHub is asking to activate copilot

### DISCUSSION THREAD

**Discussion by shuvocasanova** [community/community#19065] [Copilot Conversations]
I recently got accepted in the technical preview list. Everything was working fine till afternoon. I just started my computer and opened vs code, now gitHub copilot is giving me notification to activate copilot.

What just happened to my account. Is my account removed from GitHub copilot list

**Comment by rauzerfabiano** [comment]
Maybe this solution can help you:
https://github.com/orgs/github-community/discussions/19061#discussioncomment-2995893

The solution in the discussion says the following:
1. Go to https://github.com/github-copilot/tp_signup
2. Press the button [Continue without payment details.](https://github.com/github-copilot/signup/success)
3. In Suggestions matching public code; select Allow or Block.
4. Press Save and get started
5. Close or Reload the IDE or Code Editor

**Reply by imageaid** [reply]
After trying to setup with my payment stuff, this is what worked and got RubyMine (and my other IntelliJ IDEs) rolling again.

**Reply by rauzerfabiano** [reply]
> Depois de tentar configurar com meu material de pagamento, foi isso que funcionou e fez o RubyMine (e meus outros IDEs IntelliJ) rodar novamente.

I'm glad it was helpful
